docs/examples/rs-tutorial.rst

```diff
@@ -0,0 +1,166 @@
+Tutorial Project - Redshift
+===========================
+The project described on this page can be found in the `deepforge examples repository <https://github.com/deepforge-dev/examples>`_ on GitHub under the name **redshift-tutorial.webgmex**
+
+Pipeline Overview
+-----------------
+1. `Basic Input/Output`_
+2. `Display Random Image`_
+3. `Display Random CIFAR-10`_
+4. `Train CIFAR-10`_
+5. `Train-Test`_
+6. `Train-Test-Compare`_
+7. `Download-Train-Evaluate`_
+
+.. 6. `Visualize Predictions`_
+
+Pipelines
+---------
+
+
+Basic Input/Output
+~~~~~~~~~~~~~~~~~~
+This pipeline provides one of the simplest examples of a pipeline possible in DeepForge. Its sole purpose is to create an array of numbers, pass the array from the first node to the second node, and print the array to the output console.
+
+The **Output** operation shown is a special built-in operation that will save the data that is provided to it to the selected storage backend. This data will then be available within the same project as an artifact and can be accessed by other pipelines using the special built-in **Input** operation.
+
+.. figure:: basic-io.png
+    :align: center
+
+
+.. code-block:: python
+
+    import numpy
+
+    class GenArray():
+        def __init__(self, length=10):
+            self.length = length
+            return
+
+        def execute(self):
+            arr = list(numpy.random.rand(self.length))
+            return arr
+
+
+Display Random Image
+~~~~~~~~~~~~~~~~~~~~
+.. figure:: display-rand-img.png
+    :align: center
+
+This pipeline’s primary purpose is to show how graphics can be output and viewed. A random noise image is generated and displayed using matplotlib’s pyplot library.  Any graphic displayed using the **plt.show()** function can be viewed in the executions tab.
+
+.. code-block:: python
+
+    from matplotlib import pyplot as plt
+    from random import randint
+
+    class DisplayImage():
+        def execute(self, image):
+            if len(image.shape) == 4:
+                image = image[randint(0, image.shape[0] - 1)]
+            plt.imshow(image)
+            plt.show()
+
+Display Random CIFAR-10
+~~~~~~~~~~~~~~~~~~~~~~~
+.. figure:: display-cifar.png
+    :align: center
+
+As  with  the  previous  pipeline,  this  pipeline  simply  displays  a  single  image.   The  image  from  this pipeline, however, is more meaningful, as it is drawn from the commonly used CIFAR-10 dataset. This pipeline seeks to provide an example of the input being used in the next pipeline while providing an example of how the data can be obtained. This is important for users who seek to develop their own pipelines, as CIFAR-10 data generally serves as an effective baseline for testing and development of new CNN architectures or training processes.
```

Comment:
It might be good to make cifar-10 a link to http://www.cs.toronto.edu/~kriz/cifar.html.

Reply:
Never too many citation links. Added.